### PR TITLE
Sync with main3

### PR DESCRIPTION
**Pull request by igorosip0v**

### CODE REVIEW

File: web/legacy/api/images/verify_app.ts [hunk 1]
```diff
@@ -126,21 +126,6 @@ export const handleVerifyApp = async (
     (metadata) => metadata.verification_status === "verified",
   );
 
-  // Check if app is allowed to be app store and world app approved
```
Comment (igorosip0v) [inline web/legacy/api/images/verify_app.ts:129]: @andy-t-wang I think we should keep it, or am I missing something?

Comment (andy-t-wang) [reply web/legacy/api/images/verify_app.ts:129]: Yeah lets get rid of this of the changes in this file, I made these changes in dev-portal-2 not in main

File: web/tests/integration/team.test.ts
Comment (andy-t-wang) [inline web/tests/integration/team.test.ts:1]: lets remove this too. Looks like its just undoing the lint

File: web/tests/integration/app.test.ts
Comment (andy-t-wang) [inline web/tests/integration/app.test.ts:1]: export const IDKitBridge = memo(function IDKitBridge(props: IDKitBridgeProps) {

File: web/legacy/backend/graphql.ts
Comment (andy-t-wang) [inline web/legacy/backend/graphql.ts:1]: export const IDKitBridge = memo(function IDKitBridge(props: IDKitBridgeProps) {

File: web/legacy/api/images/verify_app.ts (hunk 1 above)
Comment (igorosip0v) [reply web/legacy/api/images/verify_app.ts]: I've revert this

File: web/tests/integration/team.test.ts
Comment (igorosip0v) [reply web/tests/integration/team.test.ts:1]: No, this is applies current linting correctly.

**Review by andy-t-wang — APPROVED** [review]

**Pull request merged**